public/assets/scss/_base.scss

```diff
@@ -1,3 +1,26 @@
+// This file is part of SpongeHome, licensed under the MIT License (MIT).
+
```

Comment:
Not sure if it does that with the partials, however that's definitely true for the main file.